### PR TITLE
Base docs content & layout improvements

### PR DESCRIPTION
**Pull request by clevinson**
This PR does the following:
- updates Getting Started page with info pertaining to `regen-devnet-1`
- update file layout for docs, and sidebar setup
- connect to Netlify for histing -- PR branch deploys work. Once this PR is merged, live docs will be rendered on [docs.regen.network](http://docs.regen.network).

### CODE REVIEW

File: docs/core-functionality.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Cosmos SDK Background
+
+As Regen Ledger is built ontop of the Cosmos SDK, much of the language and usage patterns when interacting with Regen Ledger follow directly from usage patterns and architecture laid out in the Cosmos SDK.
+
+For more information on Cosmos's work, and what it means to build an "application specific blockchain", the [Cosmos SDK Docs](https://docs.cosmos.network/master/intro/overview.html#what-are-application-specific-blockchains) is a great place to start.
+
+## Regen Ledger Overview
+
+Regen Ledger is as a single application binary that:
```
Comment (amaurymartiny) [inline docs/core-functionality.md:11]:
```suggestion
Regen Ledger is a single application binary that:
```

File: docs/docs/core-functionality.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Core Functionality
```
Comment (amaurymartiny) [inline docs/docs/core-functionality.md:1]: this file is duplicated, should be removed

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+If not connecting to a live testnet or devnet, initialize the node config for local development:
+
+```sh
+regen init [moniker]
+```
+
+
+## Running a node
```
Comment (amaurymartiny) [inline docs/getting-started.md:36]: Between `init` and `start` I believe there are still some other steps to run. We could take inspiration from https://docs.cosmos.network/master/run-node/run-node.html

**Review by amaurymartiny — APPROVED** [review]
Happy to merge if we remove the duplicate file.

edit: I pushed 12461ee myself 👍

File: docs/getting-started.md (hunk 3 above)
Comment (anilCSE) [reply docs/getting-started.md:36]: Yes, a followup pr?

**Review by anilCSE — APPROVED** [review]
Let's merge this and address TODOs in a follow up pr

**Pull request merged**